drake/solvers/Optimization.h

```diff
@@ -597,6 +598,10 @@ class DRAKEOPTIMIZATION_EXPORT OptimizationProblem {
       getLinearEqualityConstraints() const {
     return linear_equality_constraints;
   }
+  const std::list<Binding<LinearConstraint>>&
```

Comment:
My personal opinion is that we should not require a full cleanup, but we should require that new functions be compliant.  Hence this comment.  (This is a new function, right?)